README.md

```diff
@@ -55,6 +55,7 @@ Create a [personal access token](https://docs.gitlab.com/ce/user/profile/persona
 | `GL_TOKEN` or `GITLAB_TOKEN`   | **Required.** The token used to authenticate with GitLab. |
 | `GL_URL` or `GITLAB_URL`       | The GitLab endpoint.                                      |
 | `GL_PREFIX` or `GITLAB_PREFIX` | The GitLab API prefix.                                    |
+| `HTTP_PROXY` or `HTTPS_PROXY`  | HTTP or HTTPS proxy to use.                               |
```

Comment:
A couple of thoughts:

1. It looks like the [`hpagent`](https://github.com/delvedor/hpagent#usage) package supports basic authentication by embedding the username and password in the proxy URL:

   ```
   http://user:pwd@localhost:8080
   ```

   I think it's worth mentioning this somewhere in this project's README. WDYT?

1. If a user **does** put a username/password in this URL, is there a chance this could show up in the job logs? If so, we should recommend the user make the variable [masked](https://docs.gitlab.com/ee/ci/variables/#mask-a-custom-variable) to avoid this.

Reply:
Yes, I added a section in the README about proxy configuration.

I ran semantic release in debug mode and proxy configuration is not printed in the job logs.